src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -1711,21 +1721,6 @@ Map {
             "hasSingleRowEdit": Object {
               "type": "bool",
             },
-            "truncateCellText": Object {
```

Comment:
So with the removal of these props, is this forcing a major version change?

Reply:
See deprecation comment above